app/src/main/res/navigation/main_navigation.xml

```diff
@@ -246,6 +246,22 @@
             app:argType="boolean" />
     </dialog>
 
+    <dialog
+        android:id="@+id/scheduleSendBottomSheetDialog"
+        android:name="com.infomaniak.mail.ui.bottomSheetDialogs.ScheduleSendBottomSheetDialog"
+        android:label="ScheduleSendBottomSheetDialog"
+        tools:layout="@layout/bottom_sheet_schedule_send">
+        <argument
+            android:name="isAlreadyScheduled"
+            android:defaultValue="false"
+            app:argType="boolean" />
+        <argument
+            android:name="draftResource"
+            android:defaultValue="@null"
+            app:argType="string"
+            app:nullable="true" />
```

Comment:
These values probably shouln't be passed down to the bottom sheet if the bottom sheet needs to be reused elswhere. Only the caller needs to know about it and the bottom sheet must tell the caller what button was clicked and nothing more